docs/encyclopedia/event-history/go.mdx

```diff
@@ -0,0 +1,306 @@
+    "https://learn.temporal.io/courses/temporal-102/go/event-history-walkthrough/nondeterministic-workflow/nondeterministic-workflow.007.jpeg",
+    "https://learn.temporal.io/courses/temporal-102/go/event-history-walkthrough/nondeterministic-workflow/nondeterministic-workflow.008.jpeg",
+    "https://learn.temporal.io/courses/temporal-102/go/event-history-walkthrough/nondeterministic-workflow/nondeterministic-workflow.009.jpeg",
+    "https://learn.temporal.io/courses/temporal-102/go/event-history-walkthrough/nondeterministic-workflow/nondeterministic-workflow.011.jpeg",
+    "https://learn.temporal.io/courses/temporal-102/go/event-history-walkthrough/nondeterministic-workflow/nondeterministic-workflow.012.jpeg",
+    "https://learn.temporal.io/courses/temporal-102/go/event-history-walkthrough/nondeterministic-workflow/nondeterministic-workflow.013.jpeg",
+    "https://learn.temporal.io/courses/temporal-102/go/event-history-walkthrough/nondeterministic-workflow/nondeterministic-workflow.014.jpeg",
+  ]}
+  captions={[
+    "NA",
+    "Imagine the following Workflow Definition is being executed. As the Workflow executes step by step, the first line that results in a Command is the call to the `ImportSalesData` Activity. The Worker issues the `ScheduleActivityTask` Command to the Service. In this case, the execution of the Activity is successful, so the Service logs three Events to the Event History: `ActivityTaskScheduled`, `ActivityTaskStarted`, `ActivityTaskCompleted`.",
+    "Now, the Worker reaches a conditional statement which evaluates the value of a random-generated number. The random number generator happens to return the value of 84 during this execution. Since the expression evaluates to `true`, execution continues with the next line.",
+    "The next line is a request to start a Timer, so the Worker issues a Command to the Service - `StartTimer` - requesting that it starts a Timer. The Service starts the Timer, records an Event - `TimerStarted` - and then records another Event when the Timer fires - `TimerFired`.",
+    "Now imagine that the Worker happens to crash once it reaches to next line, so another Worker takes over, using Replay to restore the current state before continuing execution of the lines that follow.",
+    "The Worker then requests the Event History to replay it. Once the Worker has the Event History, the Worker determines the expected sequence of Commands needed to restore the current state. The Worker is expecting to encounter the `ScheduleActivityTask` and `StartTimer` Commands.",
+    "As the Worker executes the code during Replay, it reaches the first call to execute an Activity and creates a `ScheduleActivityTask` Command.",
+    "This Command matches the one expected based on the Event History. It's not only the right type of Command, with the same details, but it also occurs at the right position in the sequence of expected Commands. Therefore, Replay proceeds.",
```

Comment:
nit: replay doesn't _actually care_ about the details, it just result of the Command

```suggestion
    "This Command matches the one expected based on the Event History. It's not only the right type of Command, but it also occurs at the right position in the sequence of expected Commands. Therefore, Replay proceeds.",
```